frontend/app_flowy/packages/appflowy_editor/lib/src/service/internal_key_event_handlers/arrow_keys_handler.dart

```diff
@@ -322,6 +322,31 @@ ShortcutEventHandler cursorRightWordSelect = (editorState, event) {
   return KeyEventResult.handled;
 };
 
+ShortcutEventHandler cursorLeftWordDelete = (editorState, event) {
+  final nodes = editorState.service.selectionService.currentSelectedNodes;
+  final selection = editorState.service.selectionService.currentSelection.value;
```

Comment:
if the `nodes.whereType<TextNode>()` returns an empty list, it will throw an error.

```dart
ShortcutEventHandler cursorLeftWordDelete = (editorState, event) {
  final textNodes = editorState.service.selectionService.currentSelectedNodes
      .whereType<TextNode>();
  final selection = editorState.service.selectionService.currentSelection.value;
  if (textNodes.isEmpty || selection == null) {
    return KeyEventResult.ignored;
  }

  final textNode = textNodes.first;

  final startOfWord =
      selection.end.goLeft(editorState, selectionRange: _SelectionRange.word);
  if (startOfWord == null) {
    return KeyEventResult.ignored;
  }

  final transaction = editorState.transaction;
  transaction.deleteText(
    textNode,
    startOfWord.offset,
    selection.end.offset - startOfWord.offset,
  );
  editorState.apply(transaction);

  return KeyEventResult.handled;
};
```

Reply:
On it